src/pages/projects/forms/ProjectDetailsForm.tsx

```diff
@@ -179,6 +186,31 @@ const ProjectDetailsForm: FC<Props> = ({ formik, project, isEdit }) => {
               ),
             }}
           />
+          <NetworkSelector
+            value={formik.values.default_project_network}
+            project={"default"}
+            setValue={(value) =>
+              void formik.setFieldValue("default_project_network", value)
+            }
+            hasNoneOption
+            selectProps={{
+              label: "Default profile network",
+              disabled: hasNoProfiles || hasIsolatedNetworks || isEdit,
+              help: isEdit ? (
+                <>
+                  Configure networks in the{" "}
+                  <ResourceLink
+                    type="profile"
+                    value="default"
+                    to={`/ui/project/${project?.name}/profile/default`}
+                  />{" "}
+                  {" profile"}
```

Comment:
```suggestion
                  profile
```
this would create two spaces.